doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -219,6 +219,64 @@ and ``coqtop``, unless stated otherwise:
 :-batch: Exit just after argument parsing. Available for ``coqtop`` only.
 :-verbose: Output the content of the input file as it is compiled.
   This option is available for ``coqc`` only.
+:-native-compiler (yes|no|ondemand): Enable the :tacn:`native_compute`
+  reduction machine and precompilation to ``.cmxs`` files for future use
+  by :tacn:`native_compute`.
+  Setting ``yes`` enables :tacn:`native_compute`; it also causes Coq
+  to precompile the native code for future use; all dependencies need
+  to have been precompiled beforehand. Setting ``no`` disables
+  :tacn:`native_compute`; no files are precompiled.
+  Note that with ``yes``, compilation will fail if any dependency has not
+  been precompiled. ``ondemand`` enables :tacn:`native_compute`
+  but disables precompilation. All missing dependencies will be recompiled
+  every time :tacn:`native_compute` is called.
+
+  .. _native-compiler-options:
+
+  .. versionchanged:: 8.13
+
+     The default value is set at configure time,
+     ``-config`` can be used to retrieve it.
+     All this can be summarized in the following table:
+
+  .. list-table::
+     :header-rows: 1
+
+     * - ``configure``
+       - ``coqc``
+       - outcome
+       - requirements
```

Comment:
Is it worth adding a "`native_compute`" column, so that we can write "yes" or "`vm_compute`"?

Reply:
Indeed, that's much clearer than those "temporary".